assets/js/components/Tab/History/index.js

```diff
@@ -29,6 +29,12 @@ class History extends Component {
         <div><HelIcon iconName="calendar" />{' '}{ moment(cell).locale(this.props.locale).format('l') }</div>    
     )
 
+    dummyText = ()=>{
+        return (
+            <p>{this.props.intl.formatMessage({id:'app.history.yousigned.in'})}</p>
```

Comment:
It might be good to use camelCase style when there are two words at the same place like this app.history.**youSigned**.in